Engine/main/game_run.h

```diff
@@ -35,7 +35,7 @@ void GameLoopUntilNoOverlay();
 void RunGameUntilAborted();
 // Update everything game related
 void UpdateGameOnce(bool checkControls = false, IDriverDependantBitmap *extraBitmap = nullptr, int extraX = 0, int extraY = 0);
-
+// Gets current wanted game FPS, this is usually a fixed number set in script
```

Comment:
If it's maxed, then it is the measured fps. (some animations were running slowly because it was assuming 1000fps I think)